kotlin/src/main/kotlin/io/logto/sdk/core/type/TokenResponse.kt

```diff
@@ -0,0 +1,10 @@
+package io.logto.sdk.core.type
+
+data class TokenResponse(
+    val accessToken: String,
+    val refreshToken: String?,
```

Comment:
This is required?

Reply:
as i recall `openid` and `offline_access` are both required in v1? @wangsijie @simeng-li

Reply:
yep

Reply:
updated